fastlane/lib/fastlane/actions/badge.rb

```diff
@@ -110,33 +108,31 @@ def self.available_options
                                        env_name: "FL_BADGE_SHIELD_IO_TIMEOUT",
                                        description: "Set custom duration for the timeout of the shields.io request in seconds",
                                        optional: true,
-                                       is_string: false,
+                                       skip_type_validation: true, # allow integers, strings both
```

Comment:
We can actually use `type: Integer` here! A string will be converted to int with `.to_i`. We only want to use `skip_type_validation` if an int and string are both allowed and the action doesn't care about the type (which isn't very common)

```suggestion
                                       type: Integer, # allow integers, strings both
```